test/unit/enrichment/enrich-invoice-line.test.js

```diff
@@ -3,6 +3,7 @@ const mockGetSchemeCode = require('../../../app/enrichment/get-scheme-code')
 
 const scheme = require('../../mocks/scheme')
 const { SCHEME_CODE } = require('../../mocks/values/scheme-code')
+const marketingYear = 2023
```

Comment:
I think this should be below enrichInvoiceLine, but above let invoiceLine

Reply:
Agreed, moved.